egs/uw3/v1/local/chain/compare_wer.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
I guess you could change this so that CERs are printed too.
Please test your score.sh and then append the output of this script at the top of your chain recipe `run_cnn_1a.sh` along with the output of steps/info/chain_dir_info.pl.
Also please add another stage to your run.sh for chain training.